corehq/apps/translations/app_translations.py

```diff
@@ -681,7 +681,7 @@ def _update_translation_dict(prefix, language_dict, row, langs):
             language_dict.pop(lang, None)
 
     # delete anything in language_dict that isn't in langs (anymore)
-    for lang in language_dict.keys():
+    for lang in list(language_dict.keys()):
```

Comment:
Would a test for this look like uploading a translation excel that isn't in the app? Seems like it wouldn't be a hard test to write?

Reply:
https://github.com/dimagi/commcare-hq/pull/23959